node_modules/oae-principals/lib/init.js

```diff
@@ -71,7 +71,14 @@ var _ensureSchema = function(callback) {
         'PrincipalsEmailToken': 'CREATE TABLE "PrincipalsEmailToken" ("principalId" text PRIMARY KEY, "email" text, "token" text)',
 
         // Track user visits to groups they are members of
-        'UsersGroupVisits': 'CREATE TABLE "UsersGroupVisits" ("userId" text, "groupId" text, "latestVisit" text, PRIMARY KEY ("userId", "groupId"))'
+        'UsersGroupVisits': 'CREATE TABLE "UsersGroupVisits" ("userId" text, "groupId" text, "latestVisit" text, PRIMARY KEY ("userId", "groupId"))',
+
+        // 
+        'ArchiveByTenant': 'CREATE TABLE "ArchiveByTenant" ("tenantAlias" text PRIMARY KEY, "archiveId" text)',
+
+        // 
+        'DataArchive': 'CREATE TABLE "DataArchive" ("archiveId" text, "principalId" text, "resourceId" text, PRIMARY KEY ("archiveId", "principalId", "resourceId"))'
```

Comment:
I think only archiveId and principalId should form the primary key, instead of all three?